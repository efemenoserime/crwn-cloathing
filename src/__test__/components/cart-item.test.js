import React from "react";
import { cleanup, render } from "@testing-library/react";
import CartItem from "../../components/cart-item/cart-item.component";

afterEach(cleanup);

describe("<CartItem />", () => {
	const mockItem = {
		imageUrl: "https://i.ibb.co/mJS6vz0/blue-jean-jacket.png",
		price: 50,
		name: "Boots",
		quantity: 1,
	};
	it("renders CartItem correctly", () => {
		const { asFragment } = render(<CartItem item={mockItem} />);
		expect(asFragment(<CartItem />)).toMatchSnapshot();
	});
});
