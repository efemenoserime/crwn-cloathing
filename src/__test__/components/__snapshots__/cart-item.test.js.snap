// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartItem /> renders CartItem correctly 1`] = `
<DocumentFragment>
  <div
    class="cart-item"
  >
    <img
      alt="Boots"
      src="https://i.ibb.co/mJS6vz0/blue-jean-jacket.png"
    />
    <div
      class="item-details"
    >
      <span
        class="name"
      >
        Boots
      </span>
      <span
        class="price"
      >
        1 x $50
      </span>
    </div>
  </div>
</DocumentFragment>
`;
